src/main/java/teammates/ui/controller/Action.java

```diff
@@ -672,4 +676,22 @@ private boolean isMasqueradeModeRequested(AccountAttributes loggedInUser, String
     protected void excludeStudentDetailsFromResponseParams() {
         regkey = null;
     }
+
+    protected void verifyAccessibleForInstructorToFeedbackResponseComment(
```

Comment:
@damithc I think @chowyb  is right. What do you think, Should i proceed with abstraction way ?